src/components/Movie/Movie.js

```diff
@@ -5,21 +5,22 @@ import Hero from "../Hero/Hero";
 import Media from "../Media/Media";
 import MovieDescription from "../MovieDescription/MovieDescription";
 import fetchData from '../../api-calls';
+import { Redirect } from "react-router-dom";
 
 class Movie extends Component {
     constructor() {
         super();
         this.state = {
             selectedMovie: null,
             selectedMovieVideos: null,
-            singleMovieError: null,
+            singleMovieError: null
         }
     }
     componentDidMount = () => {
         fetchData(`movies/${this.props.movieId}`).then(data => {
             this.setState({selectedMovie: data.movie});
         }).catch(error => {
-            this.setState({singleMovieError: [error, this.props.movieId]});
+            this.setState({singleMovieError: error});
```

Comment:
Ahhh ya I guess we don't need that id to check which poster the error is for anymore!